example/android/build.gradle

```diff
@@ -2,30 +2,42 @@
 
 buildscript {
     ext {
-        buildToolsVersion = '30.0.2'
+        buildToolsVersion = "29.0.3"
```

Comment:
Interesting, downgrading?  expo version thing?  related to fixing the android build?

Reply:
Yes, I created again android/ios project using `expo prebuild` and that version number is prefilled by expo cli.